### DISCUSSION TITLE
iMAC 27 (late 2013)Black vertikal band overlapping screen display

### DISCUSSION THREAD

**Discussion by GG-13** [community/community#70749] [New to GitHub]
After switching from 1 TB mechanical HD to 2 TB SSD, booting with formerly built and installed OCLP Sonoma booting disk on new SSD  screen display shows black, approx. 5cm wide band over screen.  Post reinstall procedures did not help so far. (Processor 3.4 Ghz Quad Core Intel Core i5; Grapfhic Card: NIVIDIA GeForce GTX775M 2 GB, OCLP Version 1.0.1) Any reco?

**Comment by queenofcorgis** [comment]
Hi @GG-13 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Apple and Macs like: https://discussions.apple.com/community/mac. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
